lib/commands/destroy.js

```diff
@@ -22,6 +24,25 @@ module.exports = Command.extend({
   anonymousOptions: [
     '<blueprint>'
   ],
+  
+  beforeRun: function(rawArgs){
+    // merge in blueprint availableOptions
+    var blueprint;
+    try{
+      blueprint = this.lookupBlueprint(rawArgs[0]);
+      this.registerOptions( blueprint );
+    }
+    catch(e) {
+      // if the error is a SilentError, ignore
```

Comment:
@stefanpenner is this better? I'm making the assumption that `this.lookupBlueprint()` is throwing a SilentError on a nonexistent blueprint. Anything that's an actual error gets rethrown.